src/routes/tweets.route.ts

```diff
@@ -14,10 +14,12 @@ import {
 
 const tweetsRoutes = express.Router();
 
-tweetsRoutes.route("/").get(validation(getTweets), async (req, res) => {
-  const tweets = await tweetsService.getAllTweets(req);
-  res.send(tweets);
-});
+tweetsRoutes
+  .route("/")
+  .get(isAuthorized, validation(getTweets), async (req, res) => {
```

Comment:
Why did you decide close all tweets for not authorized user?  This needs to be agreed with the whole team. If this needed only for "liked" - you can check if req.user?._id ===undefined then you don't do the logic with "liked"

Reply:
Sorry, I didn't know you could view tweets without logging in